pkg/util/cert/cert.go

```diff
@@ -19,32 +19,30 @@ import (
 	cert "github.com/open-policy-agent/cert-controller/pkg/rotator"
 	"k8s.io/apimachinery/pkg/types"
 	ctrl "sigs.k8s.io/controller-runtime"
+
+	config "sigs.k8s.io/jobset/api/config/v1alpha1"
 )
 
 const (
-	serviceName             = "jobset-webhook-service"
-	secretName              = "jobset-webhook-server-cert"
-	secretNamespace         = "jobset-system"
 	certDir                 = "/tmp/k8s-webhook-server/serving-certs"
 	validateWebhookConfName = "jobset-validating-webhook-configuration"
 	mutatingWebhookConfName = "jobset-mutating-webhook-configuration"
 	caName                  = "jobset-ca"
 	caOrg                   = "jobset"
 )
 
-// dnsName is the format of <service name>.<namespace>.svc
-var dnsName = fmt.Sprintf("%s.%s.svc", serviceName, secretNamespace)
-
 //+kubebuilder:rbac:groups="",resources=secrets,verbs=get;list;watch;update
 //+kubebuilder:rbac:groups="admissionregistration.k8s.io",resources=mutatingwebhookconfigurations,verbs=get;list;watch;update
 //+kubebuilder:rbac:groups="admissionregistration.k8s.io",resources=validatingwebhookconfigurations,verbs=get;list;watch;update
 
 // CertsManager creates certs for webhooks.
-func CertsManager(mgr ctrl.Manager, setupFinish chan struct{}) error {
+func CertsManager(mgr ctrl.Manager, cfg config.Configuration, setupFinish chan struct{}) error {
+	// DNSName is <service name>.<namespace>.svc
+	var dnsName = fmt.Sprintf("%s.%s.svc", *cfg.InternalCertManagement.WebhookServiceName, *cfg.Namespace)
 	return cert.AddRotator(mgr, &cert.CertRotator{
 		SecretKey: types.NamespacedName{
-			Namespace: secretNamespace,
-			Name:      secretName,
+			Namespace: *cfg.Namespace,
```

Comment:
@tenzen-y @kannon92 I am wondering is there any specific reason that we want [to configure namespace in the Config ](https://github.com/kubernetes-sigs/jobset/blob/74e8cd088906e72d01841c5c9badbce978abdc19/api/config/v1alpha1/configuration_types.go#L31-L35)rather than taking it from the Pod's `/var/run/secrets/kubernetes.io/serviceaccount/namespace` ?

Our internal cert manager should always create certificates for the service in the same namespace where manager is deployed.

Reply:
We are technically doing that.

https://github.com/kubernetes-sigs/jobset/blob/main/api/config/v1alpha1/defaults.go#L58

This is setting the namespace based on the serviceAccount. And then we use that value for certRotation.

Reply:
Yeah, but why do we allow users to change the namespace in the Config ?

Reply:
https://github.com/kubernetes-sigs/kueue/blob/main/apis/config/v1beta1/configuration_types.go#L34

Reply:
Yeah, Kueue has similar config. I think, we should remove it, if user should not set other namespace there.
For instance, in Katib we don't allow to set that value for CertGenerator config: https://github.com/kubeflow/katib/blob/master/pkg/apis/config/v1beta1/types.go#L86-L98.
Thoughts @tenzen-y @ahg-g @danielvegamyhre ?

Reply:
IMO this is not really related to this issue/PR. 

We have a real problem where folks can't run jobset if it is installed outside of jobset-system. I'm happy to discuss this but I think it belongs as a separate issue.

Reply:
Opened #720